### ISSUE TITLE
Folder Notification Settings

### ISSUE THREAD

**Issue by NFLD99** [Aliucord/plugin-requests#524]
### Plugin Description

This plugin would add the notifications settings to the folder, this would affect all servers within the folder to match the folders notification settings screenshot below for what notification settings I'm talking about

### Go into more detail...

![Screenshot_20220112-120522](https://user-images.githubusercontent.com/24623601/149171688-ecb2389b-9b45-4b3a-aabd-b3ab6042c731.png)

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by ItzOnlyAnimal** [comment]
depending on how many servers are in the folder, would require a cooldown/rate limit or something to prevent api spam/selfbot ish

**Comment by NFLD99** [comment]
> depending on how many servers are in the folder, would require a cooldown/rate limit or something to prevent api spam/selfbot ish 

Of course, even if it takes a few minutes it would be great, having an in-app notification when it's done or progress (x/y servers set) could also be implemented

**Comment by Ushie** [comment]
It takes a few minutes to apply this manually to 100+ servers, and it's not a problem ever after so this slight inconvience I believe is fine over implementing an API excessive plugin
